<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompareWnd</class>
 <widget class="QWidget" name="CompareWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>162</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MD5校验</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
font: 9pt &quot;Consolas&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前MD5：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtCurMd5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>要与其比对的MD5：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtOtherMd5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>校验结果：</string>
   </property>
  </widget>
  <widget class="QLabel" name="stcResult">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;p style=&quot;color:#ff0000;&quot;&gt;不相同&lt;/p&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
